graphix/visualization.py

```diff
@@ -62,11 +62,11 @@ def __init__(
         local_clifford : dict
             dict specifying the local clifford for each node.
         """
-        self.G = G
+        self.g = g
```

Comment:
A longer variable or field name, such as `graph`, would be better than `g`.

Reply:
Fixed in 4780cd3b92f8c4f1569707257511c42b12984b28 .

Reply:
Fixed in 0ee3b217e1dd5db66213ddfb6b17da5a7f087df8 .